functions.Makefile

```diff
@@ -180,9 +180,10 @@ endef
 #   $1 - Path to the docker-compose file.
 #
 define docker_compose_up
-    $(ROOT)/docker_run.py --dind --sbt --root -- \
-    		--net host \
-    		docker/compose:1.21.0 -f $(ROOT)/$(1) up
+	$(ROOT)/docker_run.py --dind --sbt --root -- \
+		--net host \
+		wellcome/sbt_wrapper \
+		"project $(1)" "dockerComposeUp"
```

Comment:
why this? I intentionally bypassed sbt when I wrote this, because sbt wants to resolve the dependencies etc, before starting the containers, which makes it super slow. Eventually I just found that I was using docker-compose directly from command line cause sbt was too painful

Reply:
If the directory structure doesn't match the sbt project name (as is the case for archive_common and archive/common), this task fails.

Reply:
yeah I noticed that. I'm not sure this is less painful than having to call docker-compose directly just form archive_common. But we shall see.